RoddyCore/src/de/dkfz/roddy/Constants.java

```diff
@@ -31,6 +31,7 @@
     public static final String APP_PROPERTY_USED_LIBRARIES = "usedLibraries";
     public static final String APP_PROPERTY_EXECUTION_SERVICE_AUTH_METHOD_PWD = "password";
     public static final String APP_PROPERTY_EXECUTION_SERVICE_AUTH_METHOD_KEYFILE = "keyfile";
+    public static final String APP_PROPERTY_EXECUTION_SERVICE_AUTH_METHOD_KEYFILE_LOCATION = "executionServiceKeyfileLocation";
```

Comment:
Will there ever be another key than for the execution service? If not consider simplifying the name to `keyfileLocation`.